packages/renderless/src/cascader/index.ts

```
@@ -129,7 +129,7 @@ export const watchCheckedValue =
   }
 
 export const isEmpty = (val) => {
-  if (isNull(val)) {
+  if (type.isNull(val)) {
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)